emily/handler/src/api/handlers/limits.rs

```diff
@@ -55,7 +55,6 @@ pub async fn get_limits(context: EmilyContext) -> impl warp::reply::Reply {
         (status = 405, description = "Method not allowed", body = ErrorResponse),
         (status = 500, description = "Internal server error", body = ErrorResponse)
     ),
-    security(("ApiGatewayKey" = []))
```

Comment:
Are we also removing the limit from the set? Will anyone be able to change the limits?

Reply:
No, this is a mistake, this was meant to be the other setter. Fixing.

Reply:
Fixed.